sphinx/tutorial/cylc/furthertopics/retries.rst

```diff
@@ -0,0 +1,161 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+Retries
+=======
+
+Retries allow use to automatically re-submit tasks which have failed due to
+failure in submission or execution.
+
+
+Purpose 
+-------
+
+Retries can be useful for tasks that may occasionally fail due to external
+events, and are routinely fixable when they do - an example would be a task
+that is dependent on a system that experiences temporary outages.
+
+If a task fails, the cylc retry mechanism can resubmit it after a
+pre-determined delay. An environment variable, ``$CYLC_TASK_TRY_NUMBER``
+is incremented and passed into the task - this means you can write your
+task script so that it changes behaviour accordingly.
+
+
+Example
+-------
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/7/73/Double-six-dice.jpg
+   :width: 200px
+   :align: right
+   :alt: Two dice both showing the number six
+
+Create a new suite by running the following commands::
+
+   rose tutorial retries-tutorial
+   cd retries-tutorial
+
+You will now have a suite with a ``roll_doubles`` task which simulates
+trying to roll doubles using two dice:
+
+.. code-block:: cylc
+
+   [cylc]
+       UTC mode = True # Ignore DST
+
+   [scheduling]
+       [[dependencies]]
+           graph = start => roll_doubles => win
+
+   [runtime]
+       [[start]]
+       [[win]]
+       [[roll_doubles]]
+           script = """
+   sleep 10
+   RANDOM=$$  # Seed $RANDOM
+   DIE_1=$((RANDOM%6 + 1))
+   DIE_2=$((RANDOM%6 + 1))
+   echo "Rolled $DIE_1 and $DIE_2..."
+   if (($DIE_1 == $DIE_2)); then
+      echo "doubles!"
+   else
+      exit 1
+   fi
+   """
+
+
+Running Without Retries
+-----------------------
+
+Let's see what happens when we run the suite as it is. Open the cylc gui::
```

Comment:
I would format to ``cylc gui`` for consistency (even when it is referenced as a full command on the next line).